src/models/stores/stores.ts

```diff
@@ -78,6 +80,7 @@ class Stores implements IStores{
   ui: UIModelType;
   groups: GroupsModelType;
   class: ClassModelType;
+  docFilter: "Problem" | "Investigation" | "Unit" = "Problem";
```

Comment:
Rather than duplicating this property here and the persistenUI, it'd be better to read this property from the persistenUI.